client/my-sites/comments/comment-list/index.jsx

```diff
@@ -31,7 +38,57 @@ export class CommentList extends Component {
 		}
 	}
 
+	getNoticeContent = ( commentId, newStatus, previousStatus ) => {
+		const { translate } = this.props;
+		const options = {
+			duration: 5000,
+			id: `comment-notice-${ commentId }`,
+			isPersistent: true,
+		};
+
+		if ( 'delete' !== newStatus ) {
+			options.button = translate( 'Undo' );
+			options.onClick = () => this.setCommentStatus( commentId, previousStatus, true );
+		}
```

Comment:
this is exactly how I want undo actions to work!

that being said, this dangling boolean parameter is a bit confusing. is it entirely necessary?

(if we wanted to we could envision a Redux action to dispatch on undo which would contain the payload for setting the comment status. that might be an interesting way to have a global register of undoable actions. not for this PR or this time though)